keyboards/ogre/ergo_single/config.h

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 ctrlshiftba
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1209
+#define PRODUCT_ID      0x095D
```

Comment:
This is a pid.codes range.   I don't see this added to their list yet,  and it looks like your PR to add it was closed by you.  
https://github.com/pidcodes/pidcodes.github.com/pull/498

Until it's merged, I would prefer that a generic QMK VID/PID were used.

Reply:
👍 done. will do another PR to change them once I can do the PR in pidcodes and reference these in qmk master.